<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ValveWidget</class>
 <widget class="QWidget" name="ValveWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>515</width>
    <height>20</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>515</width>
    <height>20</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>16777215</width>
    <height>20</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QESimpleShape" name="ledStatus">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="shape">
      <enum>QSimpleShape::circle</enum>
     </property>
     <property name="colour0" stdset="0">
      <color>
       <red>240</red>
       <green>240</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>5</red>
       <green>240</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):isOpen</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-GV1</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::Fixed</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QLabel" name="lblName">
     <property name="sizePolicy">
      <sizepolicy hsizetype="MinimumExpanding" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16777215</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>SRC16-VA-PHC-GV</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblState">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>90</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getStatus</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-GV1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QELabel" name="lblFault">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>140</width>
       <height>20</height>
      </size>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):getFault</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-GV1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QEPushButton" name="btnClose">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Close</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setClose</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-GV1</string>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QEPushButton" name="btnOpen">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>50</width>
       <height>20</height>
      </size>
     </property>
     <property name="text">
      <string>Open</string>
     </property>
     <property name="variable" stdset="0">
      <string>$(group):setOpen</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=SRC01-VA-GV1</string>
     </property>
    </widget>
   </item>
   <item>
    <spacer name="horizontalSpacer_2">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeType">
      <enum>QSizePolicy::MinimumExpanding</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>5</width>
       <height>17</height>
      </size>
     </property>
    </spacer>
   </item>
   <item>
    <widget class="QESimpleShape" name="ledPSS">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="maximumSize">
      <size>
       <width>16</width>
       <height>20</height>
      </size>
     </property>
     <property name="ledType">
      <enum>QSimpleShape::Error</enum>
     </property>
     <property name="shape">
      <enum>QSimpleShape::circle</enum>
     </property>
     <property name="colour0" stdset="0">
      <color>
       <red>240</red>
       <green>5</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="colour1" stdset="0">
      <color>
       <red>5</red>
       <green>240</green>
       <blue>5</blue>
      </color>
     </property>
     <property name="variable" stdset="0">
      <string>TL2-PSS-SH:isInterlock</string>
     </property>
     <property name="variableSubstitutions" stdset="0">
      <string>group=TL2-VA-SH1</string>
     </property>
     <property name="displayAlarmStateOption" stdset="0">
      <enum>QE::Never</enum>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QELabel</class>
   <extends>QLabel</extends>
   <header>QELabel.h</header>
  </customwidget>
  <customwidget>
   <class>QEPushButton</class>
   <extends>QPushButton</extends>
   <header>QEPushButton.h</header>
  </customwidget>
  <customwidget>
   <class>QESimpleShape</class>
   <extends>QSimpleShape</extends>
   <header>QESimpleShape.h</header>
  </customwidget>
  <customwidget>
   <class>QSimpleShape</class>
   <extends>QWidget</extends>
   <header>QSimpleShape.h</header>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
